### ISSUE TITLE
Computing results on DOTA test set

### ISSUE THREAD

**Issue by swz30** [DetectionTeamUCAS/R2CNN_Faster-RCNN_Tensorflow#16]
Could you please provide an example of how do you prepare result files on test set for the DOTA server by using your pre-trained R2CNN model?

**Comment by yangxue0827** [comment]
```   
python demo_rh.py --src_folder='/PATH/TO/DOTA/IMAGES_ORIGINAL/' 
                  --image_ext='.png' 
                  --des_folder='/PATH/TO/SAVE/RESULTS/' 
                  --save_res=False
                  --gpu='0'
```
then commit the results files in tools/txt_out/  @swz30

**Comment by swz30** [comment]
Thank you @yangxue0827, 

When I run the demo_rh.py on the DOTA test images, it only provides me annotated images in the des_folder and tools/inference_results. Does not your code make text files in the format that I can submit to the DOTA evaluation server https://captain-whu.github.io/DOTA/tasks.html  ?

**Comment by yangxue0827** [comment]
results are in tools/txt_out. please read code carefully first. @swz30

**Comment by GuoleiSun** [comment]
@yangxue0827, thanks for your code. I followed what you said: using your pretrained model, test on dota test set and upload results to the server. For task1, I got following results:

mAP: 0.5031883538746729
ap of each class: plane:0.8045758220476723, baseball-diamond:0.6235978601971698, bridge:0.27558052246090115, ground-track-field:0.5051007521809955, small-vehicle:0.4260674259543338, large-vehicle:0.4180170480099051, ship:0.379523978169834, tennis-court:0.8042256307195812, basketball-court:0.6241976268091483, storage-tank:0.600279713052086, soccer-ball-field:0.4107538229795283, roundabout:0.44234079956583827, harbor:0.46493216568797485, swimming-pool:0.4108707385679589, helicopter:0.3577614017171661

As you can see, I only got 0.50, but you mentioned that we should got something around 0.575. What can be the reason?

**Comment by GuoleiSun** [comment]
Here: https://github.com/DetectionTeamUCAS/R2CNN_Faster-RCNN_Tensorflow/issues/13
You mentioned that your trained model (FasterRCNN_20180515_DOTA_v3: https://github.com/DetectionTeamUCAS/Models/tree/master/R2CNN_Faster-RCNN_Tensorflow)  can reproduce map of 0.575

**Comment by GuoleiSun** [comment]
Hi all,

I can reproduce the results by setting show_score_threshold in the cfg.py to 0.001. Please do that before you run demo_rh.py.

Thanks

**Comment by czzbb** [comment]
Helllo, how did u generate the annopath used in DOTA_devkit-master
